src/ol_infrastructure/applications/mitxpro/__main__.py

```diff
@@ -8,74 +7,27 @@
     OLVaultDatabaseBackend,
     OLVaultMariaDatabaseConfig,
 )
-from ol_infrastructure.infrastructure.operations.consul import (
-    consul_server_security_group,
-)
-from ol_infrastructure.lib.ol_types import AWSBase
+from ol_infrastructure.lib.ol_types import Apps, AWSBase
 from ol_infrastructure.lib.stack_defaults import defaults
 
 stack = get_stack()
 stack_name = stack.split(".")[-1]
 namespace = stack.rsplit(".", 1)[0]
 env_suffix = stack_name.lower()
 network_stack = StackReference(f"infrastructure.aws.network.{stack_name}")
+consul_stack = StackReference("infrastructure.operations.consul")
```

Comment:
There isn't a specific Consul stack as it's part of the `infrastructure.operations` project. It should be `operations_stack = StackReference("infrastructure.operations.{environment}.{stack_name}")` and define the proper environment to match the available stacks.